ciscripts/test.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
-systemctl status openvpn
+command -v systemctl > /dev/null && \
+	systemctl status openvpn
+command -v rc-service > /dev/null && \
+	rc-service openvpn status
```

Comment:
To avoid doubled status prints
```suggestion
if command -v systemctl > /dev/null; then
	systemctl status openvpn
elif command -v rc-service > /dev/null; then
	rc-service openvpn status
fi
```